test/tests/link_link.js

```diff
@@ -103,18 +103,24 @@ ariaTest(
       let linkLocator = By.css(ex.linkSelector);
       let linkElement = await t.context.session.findElement(linkLocator);
 
+      // Update url to remove external reference for dependable testing
+      const newUrl = t.context.url + '#test-url-change';
+      await t.context.session.executeScript(
+        function () {
+          let [selector, newUrl] = arguments;
+          document.querySelector(selector).onkeydown = function () {
+            goToLink(event, newUrl);
```

Comment:
There's no `event` here

Reply:
I think you can make this whole thing re-usable: 

```js
function onKeydownGoToLink(t, linkSelector, newUrl) {
  await t.context.session.executeScript(
    function () {
      let [selector, newUrl] = arguments;
      document.querySelector(selector).onkeydown = function (event) {
        goToLink(event, newUrl);
      };
    },
    linkSelector,
    newUrl
  )  
}
```

And called like: 

```js
const newUrl = t.context.url + '#test-url-change';
onKeydownGoToLink(t, ex.linkSelector, newUrl);
```

I've looked through the other cases and I think this meets them all.

Reply:
I appreciate the cleaner code but we can't actually apply this one to the others -- the [link example](https://w3c.github.io/aria-practices/examples/link/link.html) shows how to use aria to communicate something is a link that has onclick and onkeydown handlers, so there is a "goToLink" global function defined in the example page. The others are just regular links so we just need to replace the "href".